pvacseq/lib/pvacseq_utils.py

```diff
@@ -19,3 +23,50 @@ def iedb_to_prediction_method_lookup_dict():
 def iedb_prediction_methods():
     prediction_method_lookup_dict = iedb_to_prediction_method_lookup_dict()
     return sorted(prediction_method_lookup_dict.keys())
+
+def valid_allele_names_for_method(iedb_method):
+    #Ultimately we probably want this method to call out to IEDB but their command is currently broken
+    #curl --data "method=ann&species=human" http://tools-api.iedb.org/tools_api/mhci/
+    base_dir               = os.path.abspath(os.path.join(os.path.dirname(os.path.realpath(__file__)), '..'))
+    iedb_alleles_dir       = os.path.join(base_dir, 'iedb_alleles')
+    iedb_alleles_file_name = os.path.join(iedb_alleles_dir, "%s.tsv" % iedb_method)
+    alleles = set()
+    with open(iedb_alleles_file_name) as iedb_alleles_file:
```

Comment:
It certainly would be a worthwhile improvement (assuming it doesn't eat up too much memory) but we could probably change that when we revisit this to change it to web requests. This seems like a good enough solution for the time being!